src/Avalonia.Controls/TextBox.cs

```diff
@@ -360,6 +360,10 @@ protected override void OnKeyDown(KeyEventArgs e)
                     break;
 
                 case Key.Back:
+		            if (modifiers == InputModifiers.Control && SelectionStart == SelectionEnd)
+		            {
+			            SetSelectionForControlBackspace(modifiers);
+		            }
                     if (!DeleteSelection() && CaretIndex > 0)
```

Comment:
Nit: There seem to be some tabs in here which are screwing up alignment in the GitHub diff. Could you remove all tabs? We use all spaces. Also could you add a newline between the if statement?